src/Microsoft.Health.Dicom.Core/Features/Common/HashingHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Text;
+using EnsureThat;
+using HashDepot;
+
+namespace Microsoft.Health.Dicom.Core.Features.Common;
+internal static class HashingHelper
+{
+    /// <summary>
+    /// Gets a deterministic hash for a given value
+    /// </summary>
+    /// <param name="value"> Value to be hashed</param>
+    /// <param name="hashLength">Length of the hash to be returned. -1 means the complete hash value without trimming.</param>
+    /// <returns>Returns hashed value as string. If hashLength is provided, it will return the value to that length</returns>
+    public static string Hash(long value, int hashLength = -1)
```

Comment:
It would be good to pin some of the understanding in the resolved comments on this file with unit tests.